cmd/gen-go-sample/main.go

```diff
@@ -353,14 +354,41 @@ func (g *generator) genSample(sampConf schema_v1p2.Sample, methConf GAPICMethod)
 
 	p := g.pt.Printf
 
-	// Region tag, function signature and new client
+	// Region tag
 	argStr, err := argListStr(initInfo, g)
 	if err != nil {
 		return err
 	}
 	p("")
 	p("// [START %s]", sampConf.RegionTag)
 	p("")
+
+	// generate comments above the sample function
+	requiresNewLine := false
+
+	writeCommentLines := func(comment string) {
+		comment = strings.TrimSpace(comment)
+		writeComment(comment, nil, g)
+	}
+
+	if sampConf.Description != "" {
+		writeCommentLines(fmt.Sprintf("sample%s: %s", meth.GetName(), sampConf.Description))
+		requiresNewLine = true
+	}
+
+	for i, argName := range initInfo.argNames {
+		comment := initInfo.argTrees[i].comment
+		if comment == "" {
+			continue
+		}
+		if requiresNewLine {
+			writeCommentLines("")
+		}
+		requiresNewLine = true
+		writeCommentLines(fmt.Sprintf("%s: %s", argName, comment))
+	}
+
+	// function signature and initialize a new client
```

Comment:
nit: parallel structure between comments and within this comment

How about `// function signature and client initialization`?